### PR TITLE
ADDE-7482: update JSON file for the second conference

### PR DESCRIPTION
**Pull request by pichukov-rbi**

### CODE REVIEW

File: mobrix-conference-config.json [hunk 1]
```diff
@@ -1,433 +1,397 @@
...
+                    "icon": "people-1",
+                    "startTime": "2024-06-27T13:30:00+0200",
+                    "endTime": "2024-06-27T14:00:00+0200",
+                    "isBreak": false,
                     "speakers": [
                         {
-                            "name": "All together",
-                            "description": "Let's have lunch together",
-                            "icon": "restaurant-2"
+                            "name": "Klaudia Zemlic",
```
Comment (wzbmui) [inline mobrix-conference-config.json]:
```suggestion
                            "name": "Klaudia Zemlics",
```

File: mobrix-conference-config.json [hunk 2]
```diff
@@ -1,433 +1,397 @@
...
     "news": [
         {
             "title": "Dinner",
-            "description": "Let's go to this place: Brandauer Bierbögen.",
+            "description": "Let's go to this place: Brandauer Bierbögen",
             "icon": "restaurant-2",
             "link": "https://www.bierig.at/bierbogen/"
-        },
-        {
-            "title": "Dinner directions",
```
Comment (wzbmui) [inline mobrix-conference-config.json]: maybe keep to just correct link then?

File: mobrix-conference-config.json [hunk 3]
```diff
@@ -1,433 +1,397 @@
...
-                    "description": "Closing",
-                    "icon": "palm",
-                    "startTime": "2023-06-30T17:00:00+0200",
-                    "endTime": "2023-06-30T17:30:00+0200",
+                    "title": "Closing, celebration, pictures, winner of the photo contest",
+                    "description": "Closing ceremony and celebration of achievements",
+                    "icon": "camera",
+                    "startTime": "2024-06-28T15:00:00+0200",
+                    "endTime": "2024-06-28T15:30:00+0200",
                     "isBreak": false,
```
Comment (wzbmui) [inline mobrix-conference-config.json:383]: the breaks are kind of strange, not sure how we had them

**Review by wzbmui — APPROVED** [review]

File: mobrix-conference-config.json (hunk 3 above)
Comment (pichukov-rbi) [reply mobrix-conference-config.json:383]: also don't remember why we had them, but let's keep to not break something :)

File: mobrix-conference-config.json (hunk 2 above)
Comment (pichukov-rbi) [reply mobrix-conference-config.json]: updated

**Pull request merged**